react/Icon/Readme.md

```diff
@@ -0,0 +1,31 @@
+Provides an easy way to use SVG icons included in Cozy-UI as well
+as your custom icons.
+
+### Cozy Icons
+
+The icons should be colorable but for that, they need to have no fill
+property. Work is in progress to remove the fill attribute on our icons.
+
+```
+const icons = require('../../src/icons');
+const colors = ['red', 'blue', 'yellow', 'green']
+let i = 0;
+
+<div style={{ background: '#ccc'}}>{Object.keys(icons).map(icon => <span>
+  { icon } : <Icon icon={ icon } color={ colors[i++] } /><br/>
+</span>) }</div>
+```
```

Comment:
Two problems here:

![](https://i.imgur.com/ZrV9iaa.png)

Reply:
> paperplane icon missing

Bug from the icon, not from the docs

> only the first 4 icons

Thanks, good catch ><